tests/Orckestra.Composer.Cart.Tests/Services/Order/OrderHistoryViewService_GetOrderDetailViewModelAsync.cs

```diff
@@ -127,5 +129,141 @@ public async Task WHEN_order_is_not_customer_one_SHOULD_return_null()
             //Assert
             result.Should().BeNull();
         }
+
+        [Test]
+        [TestCase("New|Pending", new string[] { "New" })]
+        [TestCase("New|Pending", new string[] { "Pending" })]
+        [TestCase("New|Pending", new string[] { "New", "Pending" })]
+        public async Task WHEN_valid_request_EditableShipmentStates_SHOULD_return_TRUE(string editableShipmentStates, string[] cartShipmentStatuses)
```

Comment:
you also can have here parameter  = expectedIsOrderEditable and have all cases in one mehod
 [Test]
        [TestCase("New|Pending", new string[] { "New" }, true)]
        [TestCase("New|Pending", new string[] { "Pending" }, true)]
        [TestCase("New|Pending", new string[] { "New", "Pending" }, true)]
  [TestCase("New|Pending", new string[] { "Canceled" }, false)]
    [TestCase("New|Pending", new string[] { "New", "Canceled" }, false)]
[TestCase(null, new string[] { "New", "Canceled" }, false)]
[TestCase("New|Pending", null, false)]
        public async Task WHEN_valid_request_EditableShipmentStates_SHOULD_return_TRUE(string editableShipmentStates, string[] cartShipmentStatuses, bool expectedIsOrderEditable )